### ISSUE TITLE
Back and Forward buttons on questions

### ISSUE THREAD

**Issue by Malabarba** [vermiculus/sx.el#293]
Just like the help buffer has a back button, we would have that for questions. It's annoying to click on a link and lose the question you were reading.
Alternatively, we could just use a new buffer for each new question viewed, but that would lead to clutter.

**Comment by vermiculus** [comment]
I agree – forward / back buttons would be better, but maybe (in an ideal situation) we can keep an easily-navigable list of questions viewed for direct jumping.

**Comment by Asumie** [comment]
Don't know how
